apps/omg_watcher/lib/omg_watcher/api/account.ex

```diff
@@ -38,6 +38,7 @@ defmodule OMG.Watcher.API.Account do
   @doc """
   Gets all utxos belonging to the given address.
   Slow operation, compatible with security-critical.
+  TODO:  what's the actuall problem here? Storage model?
```

Comment:
Exactly - security critical Watcher uses only OMG.DB, so need to reiterate entire UTxO set matching owner in the values.
Consider replace the comment if I answered here

Reply:
Was there any work done for solving this? Maybe duplicating the values with a account as key?

Reply:
It wasn't meant to be solved I guess. In security critical watcher this function is ever needed to exit the funds.